pkg/localize/locales/en/cmd/cluster_bind.en.toml

```diff
@@ -25,19 +28,18 @@ $ rhoas cluster bind
 $ rhoas cluster bind --namespace=ns --app-name=myapp
 '''
 
-
 [cluster.bind.flag.appName]
 one = '''Name of the kubernetes deployment to bind'''
 
 [cluster.bind.flag.bindName]
 one = 'Name of the Service binding object to create when using operator'
 
-[cluster.bind.flag.forceOperator.description]
-one = '''Use ServiceBindingOperator only and fail if Operator is not installed'''
-
-[cluster.bind.flag.forceSDK.description]
-one = '''Use Service Binding SDK and skip ServiceBindingOperator even if installed on the cluster'''
+[cluster.bind.flag.useDeploymentConfig.description]
+one = '''Use DeploymentConfig to detect your application'''
```

Comment:
Just so you know, the ''' ticks is only necessary for multiline texts.

Reply:
I hate to have mental blockage so keep adding them for single strings just in case we will need multiline :D